### PR TITLE
Closes #100 add labels, save specs as JSON, improve readme

### LINKED ISSUE #100: missing labels?
### Background Information

Thanks for the awesome package.   We are making a modified version with additional variables added for our needs, and we see some labels are missing.  Is this expected?

**Example:**
https://github.com/pharmaverse/pharmaverseadam/blob/main/R/adae.R#L99 

@munoztd0 @jkovach2

### Definition of Done

No labels are missing or confirmation that it's expected.

### PR DESCRIPTION
**Pull request by manciniedoardo**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build pharmaverseadam site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new datasets occur on the "[Reference](https://pharmaverse.github.io/pharmaverseadam/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: data-raw/create_adams_data.R [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+
+library(readxl)
+library(jsonlite)
+
+# Read the .xlsx file
+specs_xlsx <- readxl::read_excel("inst/extdata/adams-specs.xlsx")
+
+specs_json <- toJSON(specs_xlsx, pretty = TRUE)
+
+write(json_data, "inst/extdata/adams-specs.json")
```
Comment (vbshuliar) [inline data-raw/create_adams_data.R:11]: Not sure where json_data variable is coming from. The JSON file looks incomplete - only 555 bytes and doesn't match the Excel data. Try using this code instead::

```
json_file <- "inst/extdata/adams-specs.json"
specs_xlsx <- readxl::read_excel("inst/extdata/adams-specs.xlsx")
specs_json <- toJSON(specs_xlsx, pretty = TRUE)
sheet_names <- excel_sheets(excel_file)
all_sheets <- lapply(sheet_names, function(sheet) {
    read_excel(excel_file, sheet = sheet)
})
names(all_sheets) <- sheet_names
json_data <- toJSON(all_sheets, pretty = TRUE)
write(json_data, file = json_file)
```

**Review by vbshuliar — CHANGES_REQUESTED** [review]

File: data-raw/create_adams_data.R (hunk 1 above)
Comment (manciniedoardo) [reply data-raw/create_adams_data.R]: done, thanks

**Review by vbshuliar — APPROVED** [review]
